karma.unit.conf.js

```diff
@@ -182,7 +194,12 @@ module.exports = function (config) {
       },
       module: {
         rules: rules
-      }
+      },
+      plugins: [
+        new webpack.DefinePlugin({
+          REACTOR_KARMA_CI_UNIT_TEST_MODE: JSON.stringify(true)
```

Comment:
This is used throughout Turbine now to control if we expose injectable functions or only export the default module with the real run-time dependencies.